lib/convict.js

```diff
@@ -349,7 +352,7 @@ var convict = function convict(def) {
      * Exports all the properties (that is the keys and their current values) as JSON
      */
     getProperties: function() {
-      return JSON.parse(JSON.stringify(this._instance));
+      return this._instance;
```

Comment:
`RegExp` is not a JSON type so it gets dropped by `JSON.stringify`.

But yeah, I hadn't thought it through from a cloning aspect, I'll come up with a different approach.